irony-completion.el

```diff
@@ -57,6 +57,19 @@
   :type '(repeat function)
   :group 'irony-completion)
 
+(defcustom irony-completion-accessibility-filter '(available deprecated)
+ "For completion, only accept candidates whose accessibility is in the list.
+
+Maps to libclang's CXAvailabilityKind:
+* https://clang.llvm.org/doxygen/group__CINDEX.html#ggada331ea0195e952c8f181ecf15e83d71af6542f250a1d21e256cd543669a49435
```

Comment:
Apparently I provided the wrong link, the right one should point to the enum, not to the NotAccessible value:
- https://clang.llvm.org/doxygen/group__CINDEX.html#gada331ea0195e952c8f181ecf15e83d71

nit: I used a start '*' because Github did not format a dash '-' correctly, could you replace the "list item" with a dash?
This is what I use in other places.